<launch>
  <node name="autopilot" pkg="asctec_autopilot" type="autopilot" respawn="false" required="true" output="screen">
    <param name="serial_port" type="string" value="/dev/ttyUSB0"/>
    <param name="serial_speed" type="int" value="57600"/>
    <param name="freq" value="20.0"/>

    <param name="enable_LL_STATUS" type="bool" value="true"/>
    <param name="interval_LL_STATUS" type="int" value="10"/>
    <param name="offset_LL_STATUS" type="int" value="2"/>

    <param name="enable_IMU_RAWDATA" type="bool" value="true"/>
    <param name="interval_IMU_RAWDATA" type="int" value="10"/>
    <param name="offset_IMU_RAWDATA" type="int" value="5"/>

    <param name="enable_IMU_CALCDATA" type="bool" value="true"/>
    <param name="interval_IMU_CALCDATA" type="int" value="1"/>
    <param name="offset_IMU_CALCDATA" type="int" value="0"/>

    <param name="enable_RC_DATA" type="bool" value="true"/>
    <param name="interval_RC_DATA" type="int" value="4"/>
    <param name="offset_RC_DATA" type="int" value="1"/>

    <param name="enable_CONTROLLER_OUTPUT" type="bool" value="true"/>
    <param name="interval_CONTROLLER_OUTPUT" type="int" value="10"/>
    <param name="offset_CONTROLLER_OUTPUT" type="int" value="0"/>

    <param name="enable_GPS_DATA" type="bool" value="true"/>
    <param name="interval_GPS_DATA" type="int" value="4"/>
    <param name="offset_GPS_DATA" type="int" value="2"/>

    <param name="enable_GPS_DATA_ADVANCED" type="bool" value="true"/>
    <param name="interval_GPS_DATA_ADVANCED" type="int" value="4"/>
    <param name="offset_GPS_DATA_ADVANCED" type="int" value="3"/>

    <param name="enable_CONTROL" type="bool" value="true"/>
    <param name="interval_CONTROL" type="int" value="1"/>
    <param name="offset_CONTROL" type="int" value="0"/>

    <remap from="/autopilot/CTRL_INPUT" to="/CtrlInput"/> 
  </node>
</launch>
